src/ngx_http_lua_phase.c

```diff
@@ -107,4 +107,21 @@ ngx_http_lua_inject_phase_api(lua_State *L)
     lua_setfield(L, -2, "get_phase");
 }
 
+#ifndef NGX_LUA_NO_FFI_API
```

Comment:
committers will find lots of style problems, maybe we should add `ngx-releng`, `lua-releng`,`reindex` to travis CI, in order to save time.